src/driver.rs

```diff
@@ -0,0 +1,911 @@
+#![deny(missing_docs)]
```

Comment:
The top of this file should have a two-line comment giving the terse copyright and licensing information.